### PR TITLE
[ADD] financial indicators calculation

### PR DESCRIPTION
**Pull request by Danisan**
This module adds the option to update UF, UTM, USD, and EUR from SBIF (now called CMF.cl).

### CODE REVIEW

**Comment by max3903** [comment]
wait wait...! checking it! needs some reviews

Aha :) #12

**Comment by max3903** [comment]
@Danisan Sabes donde esta la dependencia?

**Comment by Danisan** [comment]
> @Danisan Sabes donde esta la dependencia?

Cual @max3903 ?

**Comment by Danisan** [comment]
@max3903 the tests passed. Would you please test? Regarding the name... think is the same, but the module is already at odoo apps with this name. What do you suggest?

**Comment by max3903** [comment]
@Danisan

* I am talking about `webservices_generic`
* My suggestion would be to remove the module from the other repo and maintain it here collaboratively.
* You pushed your branch directly to the OCA repo. Can you push it to your account or organization?

**Comment by Danisan** [comment]
> * webservices_generic

It is  [here](https://github.com/odoo-chile/webservices_generic), but in the new version there is no need of this. It was just to hold the connection parameters. In the new version it has been replaced by data in ir.config_parameter.

File: l10n_cl_financial_indicators/data/res.currency.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+"id","rounding","name","position","symbol","currency_unit_label"
+"base.CLP","1.0","CLP","before","CLP","Peso"
+"base.EUR","0.01","EUR","before","€","Euros"
+"base.UF","0.01","UF","after","UF","Unidad de Fomento"
```
Comment (max3903) [inline l10n_cl_financial_indicators/data/res.currency.csv]:
```suggestion
"l10n_cl_financial_indicators.UF","0.01","UF","after","UF","Unidad de Fomento"
```

File: l10n_cl_financial_indicators/data/res.currency.csv [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+"base.USD","0.01","USD","before","USD","Dollars"
+"base.UTM","0.01","UTM","after","UTM","UTM"
```
Comment (max3903) [inline l10n_cl_financial_indicators/data/res.currency.csv]:
```suggestion
"l10n_cl_financial_indicators.UTM","0.01","UTM","after","UTM","UTM"
```

File: l10n_cl_financial_indicators/query0.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+UPDATE decimal_precision set digits = 14 where name = 'Currency';
```
Comment (max3903) [inline l10n_cl_financial_indicators/query0.sql]: Missing newline.

File: l10n_cl_financial_indicators/query1.sql [hunk 4]
```diff
@@ -0,0 +1 @@
+UPDATE res_currency set rounding = 0.00000000000001 where name in ('UF', 'UTM');
```
Comment (max3903) [inline l10n_cl_financial_indicators/query1.sql]: Missing newline.

File: l10n_cl_financial_indicators/data/res.currency.csv (hunk 1 above)
Comment (Danisan) [reply l10n_cl_financial_indicators/data/res.currency.csv]: same for UTM?

File: l10n_cl_financial_indicators/query1.sql (hunk 4 above)
Comment (Danisan) [reply l10n_cl_financial_indicators/query1.sql]: fixed in f9f031a9

File: l10n_cl_financial_indicators/query0.sql (hunk 3 above)
Comment (Danisan) [reply l10n_cl_financial_indicators/query0.sql]: fixed in f9f031a9

**Review by max3903 — CHANGES_REQUESTED** [review]
@Danisan Can you split the README.rst in a readme folder and add the `maintainers` and `development_status` keys in the manifest?

Ref: https://github.com/OCA/maintainer-tools/tree/master/template/module

**Comment by Danisan** [comment]
> @Danisan
> 
> * I am talking about `webservices_generic`
> * My suggestion would be to remove the module from the other repo and maintain it here collaboratively.
> * You pushed your branch directly to the OCA repo. Can you push it to your account or organization?

OK @max3903 I will push from mine.

**Comment by Danisan** [comment]
This thread will be closed then, to make the PR from our repo.

**Pull request closed without merging**